### PR TITLE
Make it easier to run more tests

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: tests/kvlds-s3/test_kvlds.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ TMPDIR=`pwd`/tmp
 SOCKS3=$TMPDIR/sock_s3
 SOCKL=$TMPDIR/sock_lbs
 SOCKK=$TMPDIR/sock_kvlds
-REGION=us-west-2
+REGION=${REGION:-us-west-2}
```
Comment (gperciva) [inline tests/kvlds-s3/test_kvlds.sh:13]: I'm not certain why some REGIONs were hard-coded to `us-east-1`, while others were `us-west-2`.

Comment (cperciva) [reply tests/kvlds-s3/test_kvlds.sh:13]: I use us-east-1 as a default, but at some point I wanted to test with a "closer" region.  I think.

File: BUILDING [hunk 2]
```diff
@@ -6,6 +6,33 @@ which includes the Software Development Utilities and Threads options.  If it
...
+    ACCESS_KEY_ID=...
+    ACCESS_KEY_SECRET=...
+(with no leading spaces)
+
+- tests/s3: you must:
+  1) have a ~/.s3/aws.key file which contains the access key.
+  2) have a unique S3 bucket name in AWS.
+  3) specify the bucket name as the BUCKET environment variable.  For example,
+
+      $ BUCKET=kivaloo-test-john-smith make test
```
Comment (cperciva) [inline BUILDING:26]: Do you want to mention REGION here too?

Comment (gperciva) [reply BUILDING:26]: I considered it, but then I figured that the command line would look too ugly.

That said, I guess that we should mention REGION somewhere, so I'll do that.

**Comment by gperciva** [comment]
Rebased to mention REGIONs.  I deliberately used different regions in the two examples.

**Pull request merged**